backtracking/subset_sum.cpp

```diff
@@ -47,6 +53,30 @@ uint64_t number_of_subsets(int32_t sum, const std::vector<int32_t> &in_arr) {
     }
     return count_of_subset;
 }
+
+//2nd Method
```

Comment:
Missing proper documentation. Check the [typical structure of a program](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING#typical-structure-of-a-program) for more information. 🙂